lib/propagation/util.js

```diff
@@ -23,3 +23,39 @@ function getPropagationContext(context = {}) {
     customContext: context.traceContext,
   };
 }
+
+exports.objToString = objToString;
+
+// pass this util an object, return as string formatted to go into a trace header
+// joinChar separates fields from each other, keyChar separates keys from values
+// function is written to place a joinChar at the beginning of the string
+function objToString(obj, joinChar = ",", keyChar = "=") {
+  let string = "";
+  const keys = Object.keys(obj) || {};
```

Comment:
Just peeking at this, realized this should fallback to an empty array, not an empty Object.
```suggestion
  const keys = Object.keys(obj) || [];
```